Source/Core/Core/PowerPC/CachedInterpreter/CachedInterpreter.cpp

```diff
@@ -115,24 +129,15 @@ s32 CachedInterpreter::HLEFunction(PowerPC::PowerPCState& ppc_state,
   return sizeof(AnyCallback) + sizeof(operands);
 }
 
-s32 CachedInterpreter::WritePC(PowerPC::PowerPCState& ppc_state, const WritePCOperands& operands)
-{
-  const auto& [current_pc] = operands;
-  ppc_state.pc = current_pc;
-  ppc_state.npc = current_pc + 4;
-  return sizeof(AnyCallback) + sizeof(operands);
-}
-
 s32 CachedInterpreter::WriteBrokenBlockNPC(PowerPC::PowerPCState& ppc_state,
-                                           const WritePCOperands& operands)
+                                           const WriteBrokenBlockNPCOperands& operands)
 {
   const auto& [current_pc] = operands;
   ppc_state.npc = current_pc;
   return sizeof(AnyCallback) + sizeof(operands);
 }
 
-s32 CachedInterpreter::CheckFPU(PowerPC::PowerPCState& ppc_state,
-                                const ExceptionCheckOperands& operands)
+s32 CachedInterpreter::CheckFPU(PowerPC::PowerPCState& ppc_state, const CheckFPUOperands& operands)
```

Comment:
lint fixes should happen in the commit where the problem started, not the latest commit.